iocore/net/unit_tests/sni_conf_test.yaml

```diff
@@ -0,0 +1,5 @@
+sni:
+- fqdn: allports.com
+- fqdn: someport.com:1-433
+- fqdn: someport.com:8080-65535
```

Comment:
Maybe add a distinguishing attribute under each of these to make sure that each range is matching to the expected set of attributes. Maybe use `http2` since that's a simple on/off value?